lib/std/os/linux/io_uring.zig

```diff
@@ -2089,7 +2090,8 @@ test "sendmsg/recvmsg" {
     try testing.expectEqual(linux.io_uring_cqe{
         .user_data = 0x22222222,
         .res = buffer_recv.len,
-        .flags = 0,
+        // ignore IORING_CQE_F_SOCK_NONEMPTY since it is set non-deterministically
```

Comment:
`since it is set non-deterministically` reads abit odd. I guess you mean that it creates non-determinism in execution. Can you shed some brief light of what is becoming non-deterministic here?

Also: Why are the comments for the same `flags` change different?

reference https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/io_uring.h#n323

Reply:
For `accept/connect/send/recv`, I'm am seeing the flag being set 100% of the time on my system, but it appears to be reset 100% of the time on other systems, so it is possibly due to a kernel version/configuration difference.

However, for `sendmsg/recvmsg`, by `since it is set non-deterministically`, I mean that the value of the flag depends on some non-deterministic process such as kernel threads or unrelated network traffic.  For this test, I'm seeing the flag being set 96%-97% of the time on my system.